### PR TITLE
Some updates for version 1.0.

### PR DESCRIPTION
**Pull request by moorepants**
- Move's all the install and development info from the README into the documentation.
- Adjusts copyright statements to focus on the "cyipopt developers" instead of a single author.

### CODE REVIEW

**Comment by moorepants** [comment]
@brocksam This is ready to be reviewed. Mostly just docs reorganizing and clean up.

File: AUTHORS [hunk 1]
```diff
@@ -1,10 +1,12 @@
 Amit Aides <amitibo@tx.technion.ac.il>
+Anders Andreasen;58475535+andr1976@users.noreply.github.com
```
Comment (moorepants) [inline AUTHORS:2]:
```suggestion
Anders Andreasen
```

**Review by brocksam — APPROVED** [review]

**Comment by brocksam** [comment]
All looks good to me. Please merge, @moorepants, when you're ready. Then I'll fix merge conflicts with PR #86 and submit that for review/merge afterwards.

**Comment by moorepants** [comment]
Thanks!

**Pull request merged**